Bottom: f850f049d9e1f9688cf437bed6e3059d0b444588
Top:    6886dba704be7c3aaf8393b23d04edbaccbdac1e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-02-11 00:42:59 +0000

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index af5aa3e..70cc17f 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3090,7 +3090,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
